src/yukon/backend/src/devserv/description/example/example_sys_description.json

```
@@ -0,0 +1,152 @@
+{
+        "bus": {
+                "resilience": 2,
+                "name": "socketcan0",
+                "protocol": "CANFD"
```

```suggestion
                "transport": "CAN"
```

1. "Protocol" is too generic.
2. UAVCAN does not distinguish CAN and CAN FD.